pypit/arcimage.py

```diff
@@ -61,7 +61,7 @@ class ArcImage(processimages.ProcessImages, masterframe.MasterFrame):
       Final output image
     """
     # Keep order same as processimages (or else!)
-    def __init__(self, file_list=[], spectrograph=None, settings=None, det=1, setup=None, sci_ID=None,
+    def __init__(self, file_list=[], spectrograph=None, settings=None, user_settings = None, det=1, setup=None, sci_ID=None,
```

Comment:
I don't think `user_settings` is going to last.
But we can have it here for now.

Reply:
Okay we should discuss this sooner rather than later, because we need to converge on something stable. It is too difficult to develop when things are changing so fast.